node/clients/AdServer.ts

```diff
@@ -27,6 +32,20 @@ class AdServer extends ExternalClient {
       ...body,
     })
   }
+
+  public getSponsoredBanners(
+    body: AdServerSponsoredBannersRequest
+  ): Promise<AdServerSponsoredBannersResponse> {
+    if (process.env.VTEX_APP_LINK) {
+      // eslint-disable-next-line no-console
+      console.log(JSON.stringify(body, null, 2))
```

Comment:
I think adding the endpoint we are calling (`/api/ads/v1/ads/banner/query`) would help too.

Reply:
You mean to print the endpoint too?

Reply:
yep!
Now there are two different endpoints. so It would be important to be able do tell the difference between then